include/ardpolyfills/Arduino.h

```diff
@@ -16,10 +16,40 @@
  *
  */
 
-#ifndef Arduino_h
-#define Arduino_h
+#ifndef SMARTCAR_EMUL_ARDUINO_HXX
+#define SMARTCAR_EMUL_ARDUINO_HXX
+#include <Thread>
+#include <chrono>
+#include <iostream>
+#include <stdint.h>
+
+typedef enum { RIGHTMOSTBIT = 0, LEFTMOSTBIT = 1 } SignificantBit;
+
+typedef enum { HIGH = true, LOW = false } digitalValue;
```

Comment:
Same as above; can even use `bool` as underlying type:
Also, incorrect casing: type names should be PascalCase
```cpp
enum DigitalValue : bool {
    LOW,
    HIGH
};
```